compliance/google/unlabeled_resources/README.md

```diff
@@ -4,6 +4,12 @@
 
 Find all Google cloud resources(disks, images, instances, snapshots, buckets, vpnGateways) missing any of the user provided labels with the option to update the resources with the missing labels.
 
+## Functional Details
+
+- The policy leverages the Google Cloud API to retrieve a list of all labelable resources across Google Cloud Projects.
+- Using the 'List of labels' parameter, the policy identifies all resources that are missing the label keys specified by the user.
```

Comment:
To match with the other 2 functional details I wonder if could be better put something like:

`The policy identifies all resources that are missing the label keys specified by the user using the 'List of labels' parameter.`

Reply:
@juanestrahcl the reason we put it in this format is to keep it in line with the other two Untagged Resources policies for AWS/Azure, see https://github.com/flexera-public/policy_templates/tree/master/compliance/aws/untagged_resources#functional-details 

I understand your point though, maybe we can look to do this in a future update, across all 3 Untagged/Unlabeled Resources policies.